presto-docs/src/main/sphinx/release/release-0.213.rst

```diff
@@ -0,0 +1,93 @@
+=============
+Release 0.213
+=============
+
+General Changes
+---------------
+
+* Fix split scheduling back pressure when plan contains colocated join. Previously, splits
+  for the second and subsequent scan nodes (in scheduling order) were scheduled continuously
+  until completion, rather than pausing due to sufficient pending splits.
+* Fix query execution failure or indefinite hang when some buckets are completely filtered out
+  during bucket-by-bucket execution.
+* Fix concurrent lifespan when grouped execution is enabled. Previously, when grouped
+  execution is enabled, it will always run with one lifespan per task (ignoring the
+  ``concurrent_lifespans_per_task`` session property).
+* Fix :func:`ST_Distance` function to return ``NULL`` if any of the inputs are an
+  empty geometry as required by the SQL/MM specification.
+* Fix returning an excessively large number of rows to clients in a single response.
+* Enhance ``SHOW STATS`` to support ``IN`` and ``BETWEEN`` predicates in the
+  ``WHERE`` condition of the ``SELECT`` clause.
+* Do not allow setting invalid property values with :doc:`/sql/set-session`.
+* Disable stats calculator by default as it can cause a planning failure for
+  certain complex queries. It can be enabled with the ``experimental.enable-stats-calculator``
+  configuration property or the ``enable_stats_calculator`` session property.
+* Avoid making wild guesses when estimating filters for joins. Previously, if nothing
+  was known about the filter, a ``0.9`` coefficient was applied as a filter factor.
+  Now, if nothing is known about a filter, the estimate will be unknown. A ``0.9``
+  coefficient will be applied for all additional conjuncts if at least a single
+  conjunct can be reasonably estimated.
+* Improve inference of predicates for inner joins.
+* Improve ``EXPLAIN ANALYZE`` output by including CPU time and scheduled time.
+* Include stats and cost estimates in textual plans created on query completion.
+* Remove transaction from explain plan for indexes.
+* Add ``max_drivers_per_task`` session property, allowing users to limit concurrency by
+  specifying a number lower than the system configured maximum. This can cause the
+  query to run slower and consume fewer resources.
+* Add ``join-max-broadcast-table-size`` configuration property and
+  ``join_max_broadcast_table_size`` session property to control the maximum
+  estimated size of a table that can be broadcast for a join (:issue:`11667`)/
```

Comment:
`/` at the end of the line should be `.`

Reply:
clarify that this is when `join_distribution_type='AUTOMATIC'`. If you set broadcast join explicitly it will still work as before.